.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Roles of a PR: Author, Reviewer, and Assignee
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
```

Comment:
Yes, I meant `Review - Needed`.  I was told (forgot by whom) that was what this label meant.  If not, we should document what exactly this label means (the current docs just has a link another doc without really defining this label)